### ISSUE TITLE
accessibility with Orca screen reader

### ISSUE THREAD

**Issue by guyster104** [mpoletiek/py3-CHIRP#24]
I am a blind user of Chirp, and I would like to use it to program my Baofeng UB5R radio. The problem I have is that I cannot access the settings or memory programming, using the Orca screen reader. I can access the menu bar, and can download an image from the radio, but I cannot interact with the various settings. Orca will say memories tab, but pressing tab or any of the arrow keys yields silence.

**Comment by mpoletiek** [comment]
Apologies, but this repo isn't really maintained. It's a fork of an upstream package which the members refuse to get working on Python 3. Unfortunately I own this repo, but don't have the time or radios to test and maintain it.

**Comment by mpoletiek** [comment]
Repo not maintained
